Makefile

```
@@ -10,20 +10,11 @@ serve:
-# Also make sure to clean up the "testing" db group 
-# in turso as that will have any user dbs that were 
+# Also make sure to clean up the "testing" db group
+# in turso as that will have any user dbs that were
 # created even locally.
 delete-local-data:
 	sudo rm -rf ./libsql
-	sudo rm -rf ./meili_data
 	sudo rm -rf ./apps/api/local.db*
 
-.PHONY: serve build prod cleanup
+.PHONY: local-db serve build delete-local-data
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)